File: src/codes/Users_Manual/gain/plane_1.cs
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp;
~var nx = 0.0f;
~var ny = 0.0f;
~var nz = 1.0f;
var g = new Plane(new Vector3(nx, ny, nz))
                .WithIntensity(0xFF)
                .WithPhaseOffset(0x00);

File: src/codes/Users_Manual/gain/plane_1.cpp
//~#include<autd3.hpp>
//~#include <limits>
//~int main() {
//~const auto nx = 0.0;
//~const auto ny = 0.0;
//~const auto nz = 1.0;
auto g = autd3::gain::Plane(autd3::Vector3(nx, ny, nz))
             .with_intensity(0xFF)
             .with_phase_offset(0x00);
//~return 0; }